peer.go

```diff
@@ -1089,8 +1063,23 @@ out:
 			isLinkUpdate = p.handleError(msg)
 
 		case *lnwire.ChannelReestablish:
-			isLinkUpdate = true
 			targetChan = msg.ChanID
+			isLinkUpdate = p.isActiveChannel(targetChan)
+
+			// If we failed to find the link in question, and the
+			// message received was a channel sync message, then
+			// this might be a peer trying to resync closed channel.
+			// In this case we'll try to resend our last channel
+			// sync message, such that the peer can recover funds
+			// from the closed channel.
+			if !isLinkUpdate {
+				err := p.resendChanSyncMsg(targetChan)
+				if err != nil {
+					// TODO(halseth): send error to peer?
+					peerLog.Errorf("resend failed: %v",
```

Comment:
Nit: I know this was pre-existing, but would be nice to add additional context to this log. Should also start capitalized.